sml_builder/templates/methods.html

```diff
@@ -78,14 +119,20 @@ <h2>{{
   {{
     onsTable(methodsTable)
   }}
-
+{% else %}
+  <p class="ons-panel__error" id="search-alert">
+    <strong>No ready to use methods found, please enter another search term.</strong>
```

Comment:
Could we tweak the wording to be:
No methods found under Methods Ready to Use, please try another search term.